src/bin/huak/main.rs

```diff
@@ -2,19 +2,23 @@
 //!
 //! Huak implements a cli application with various subcommands.
 use clap::{self, ArgMatches};
-use huak::errors::{CliError, CliResult};
+use huak::errors::{CliError, CliErrorCode, CliResult};
 
 mod commands;
 
 /// Launch Huak's cli process.
-pub fn main() -> CliResult {
+pub fn main() {
     let args = commands::args();
 
-    run(args.get_matches())
+    let res = run(args.get_matches());
+    match res {
+        Ok(_) => println!("Exited correctly."),
```

Comment:
```suggestion
        Ok(_) => (),
```
This is probably too busy. Part of the design goals for huak is to create a snappy, responsive, and unobtrusive package manager. So if we don't **need** to print, I don't think we should. Exiting successfully is exiting without an error.